.github/workflows/main.yml

```diff
@@ -0,0 +1,91 @@
+name: GameCI 🎮
+
+on: [pull_request]
+
+jobs:
+  test:
+    name: Run Unit Tests 🧪
+    runs-on: ubuntu-latest
+    
+    steps:
+      # Checkout
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          lfs: false
+
+      # Cache
+      - uses: actions/cache@v3
+        with:
+          path: Library
```

Comment:
This path should be `sample-app/Library` and the key should be `Library-${{ hashFiles('sample-app/Assets/**', 'sample-app/Packages/**', 'sample-app/ProjectSettings/**') }}`